python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -172,5 +172,19 @@ def optimize(model):
                         module.SelfAttention,
                         chatglm_attention_forward
                         )
-
+    elif "falcon" in model.config._name_or_path:
+        modeling_module_name = model.__class__.__module__
+        module = importlib.import_module(modeling_module_name)
+        if "RWForCausalLM" in model.config.architectures:
```

Comment:
Add a comment to say this is for 7b/40b model and the elif part is for 180b model?